radioactive-decay/info.yml

```diff
@@ -0,0 +1,15 @@
+---
+uid: radioactive_decay 
+title: Radioactive decay 
+prerequisites:
+  - basic statistics and probability
+  - undergraduate level  
+learning_objectives:
+  - Basic understanding of radioactivity
```

Comment:
This is a broad learning objective, perhaps more specificity?  (also a contrib/style guide issue)